bin/parameter_tuning.R

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env Rscript
+# TODO
+# Allow for user to input their own annotation dataset
+# REQUIREMENTS
```

Comment:
I assume you are the author and no further licenses etc need to be added?

Reply:
I've worked on it, @sruthipsuresh wrote it originally, so no additional licenses needed.